doc/reduce-traffic.md

```diff
@@ -19,8 +19,8 @@ This is *not* a hard limit; only a threshold to minimize the outbound
 traffic. When the limit is about to be reached, the uploaded data is cut by no
 longer serving historic blocks (blocks older than one week).
 Keep in mind that new nodes require other nodes that are willing to serve
-historic blocks. **The recommended minimum is 144 blocks per day (max. 144MB
-per day)**
+historic blocks.
+**The current recommended minimum is 72 full blocks (by size "on the wire" per day)**
```

Comment:
Nit: (to address the feedback by @wodry )

**The current recommended minimum is `-maxuploadtarget=$recommendedMinimum`, assuming an average serialized block size of 2MB.**